source/faas-automated-testing/faas-automated-testing.rst

```diff
@@ -213,6 +213,9 @@ Run the function locally from the same directory where the template.yaml resides
   sam build
   sam local invoke --parameter-overrides "MongoDbUri=${MONGODB_URI}"
 
+*NOTE* "127.0.0.1" in the MONGODB_URI MUST be replaced with "host.docker.internal" to test
```

Comment:
The [go1.x runtime](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html) only supports x86_64 architectures, which is fine for the CI since RHEL 8.0 and Ubuntu 20.04 are amd64. However, the "host.docker.internal" solution does not appear work on M1. We could use a [provider.al2](https://aws.amazon.com/blogs/compute/migrating-aws-lambda-functions-to-al2/) solution, but this adds additional complexity in that we would have to maintain a custom runtime and maintain both a amd64 bin and arm64 bin.

It _could_ be beneficial to suggest that if the "host.docker.internal" solution does not work, then run a server in a workstation or in a [bridged docker container](https://docs.docker.com/network/bridge/). And use those IP as the MONGODB_URI.

Reply:
I've rebased and added the suggested note about the bridged container.